### PR TITLE
docs: update jest.mock for 'NativeAnimatedHelper' in React Navigation…

### PR DESCRIPTION
**Pull request by raychanks**

### CODE REVIEW

File: website/docs/ReactNavigation.md [hunk 1]
```diff
@@ -165,6 +165,9 @@ import AppNavigator from '../AppNavigator';
 // Silence the warning https://github.com/facebook/react-native/issues/11094#issuecomment-263240420
```
Comment (thymikee) [inline website/docs/ReactNavigation.md:165]:
```suggestion
// Silence the warning https://github.com/facebook/react-native/issues/11094#issuecomment-263240420
// Use with React Native <= 0.63
```

File: website/docs/ReactNavigation.md [hunk 2]
```diff
@@ -163,8 +163,12 @@ import { render, fireEvent } from '@testing-library/react-native';
 import AppNavigator from '../AppNavigator';
 
 // Silence the warning https://github.com/facebook/react-native/issues/11094#issuecomment-263240420
+// Use with React Native <= 0.63
 jest.mock('react-native/Libraries/Animated/src/NativeAnimatedHelper');
 
+// Use the following mock instead if you're running React Native@0.64.0 or higher
```
Comment (thymikee) [inline website/docs/ReactNavigation.md:169]:
```suggestion
// Use this instead with React Native >= 0.64
```

**Review by thymikee — APPROVED** [review]
Thanks!

**Comment by raychanks** [comment]
Thanks, feeling good for contributing

**Pull request merged**